instagram/src/App.js

```diff
@@ -0,0 +1,23 @@
+import React, { Component } from 'react';
+import PostPage from './components/PostContainer/PostPage';
+import authenticateHOC from './components/Authentication/Authenticate';
+
+
+class App extends Component {
+  constructor(){
+    super();
+    this.state = {}
+  }  
+
+
+  render() {
+    return (
+      <div className="App">
+        <PostPage />
```

Comment:
You should probably extract the header component out of PostPage and render it here.  There's not a lot of a point to rendering a single master component from what should be your single master component.  Like you could literally save a step here, delete App.js, render PostPage in index.js instead of App.js and nothing about your application would change.